common/log.c

```diff
@@ -1140,3 +1139,32 @@ getLogFile(char *replybuf, int bufsize)
 
     return replybuf;
 }
+
+/**
+ * Returns formatted datetime for log
+ * @return
+ */
+char *
+getFormattedDateTime(char *replybuf, int bufsize)
+{
+    char buf_datetime[21];
+    char buf_millisec[4];
+    char buf_timezone[6];
+
+    struct tm *now;
+    struct timeval tv;
+    int millisec;
+
+    gettimeofday(&tv, NULL);
+    now = localtime(&tv.tv_sec);
+
+    millisec = lrint(tv.tv_usec / 1000.0);
```

Comment:
Do we need to use `lrint()` for this?

According to IEEE Std 1003.1 [<sys/time.h>](https://pubs.opengroup.org/onlinepubs/009695399/basedefs/sys/time.h.html) and [<sys/types.h>](https://pubs.opengroup.org/onlinepubs/009695399/basedefs/sys/types.h.html), `tv.tv_sec` is *a signed integer type capable of storing values at least in the range [-1, 1000000]*. So you should be able to round the value with `millisec = (tv.tv_usec + 500) / 1000`. 

This is also (I think) the only dependency on `-lm`

Reply:
Indeed. Fixed, thanks!